libraries/ms-drivers/src/ads1259_adc.c

```diff
@@ -1,6 +1,7 @@
 #include "ads1259_adc.h"
-
+#ifndef MPXE
```

Comment:
so we recently changed this structure in MPXE, we're now gonna try to just use #ifdefs within the original code since that should make it cleaner and more obvious what's going on in MPXE vs not in MPXE, so rather than having the whole file in a #else / #endif, just take the MPXE specific parts and add them to the original code with a #ifdef / #endif. There should be examples in drivers on the mpxe branch. Sorry about this, we're still ironing out the details.